components/salesforce_rest_api/actions/find-records/find-records.mjs

```diff
@@ -40,8 +40,8 @@ export default {
     } = this;
     const response = await this.salesForceRestApi.getRecords(
       sobjectType, {
-        fields: fields.join(","),
-        ids: ids.join(","),
+        fields: Array.isArray(fields) && fields.join(","),
+        ids: Array.isArray(ids) && ids.join(","),
```

Comment:
I think that if these fields are required you need a fallback in case they were set as strings
```suggestion
        fields: Array.isArray(fields) && fields.join(",") || fields,
        ids: Array.isArray(ids) && ids.join(",") || ids,
```